### PR TITLE
fix_d3a_worker_name

### PR DESCRIPTION
**Pull request by hannesdiedrich**

### CODE REVIEW

**Review by faizan2590 — APPROVED** [review]

File: src/d3a/d3a_core/d3a_jobs.py [hunk 1]
```diff
@@ -159,7 +159,7 @@ def main():
                                          retry_on_timeout=True)):
         Worker(
             [get_simulation_queue_name()],
-            name='simulation.{}.{:%s}'.format(getpid(), now()), log_job_description=False
+            name=f'simulation.{getpid()}.{now().timestamp()}', log_job_description=False
```
Comment (spyrostz) [inline src/d3a/d3a_core/d3a_jobs.py:162]: In order to make it a bit more inventive I would recommend to use time() in order to get milliseconds too. That way we can be almost certain that no 2 workers will start concurrently.

Comment (hannesdiedrich) [reply src/d3a/d3a_core/d3a_jobs.py:162]: `.timestamp()` uses milliseconds too, this was the whole point. :-)

Comment (spyrostz) [reply src/d3a/d3a_core/d3a_jobs.py:162]: all good, thanks for explaining!

**Review by spyrostz — APPROVED** [review]
LGTM

**Pull request merged**